### DISCUSSION TITLE
Add serial number or row index column type

### DISCUSSION THREAD

**Discussion by Stridesdata** [Power-Components/livewire-powergrid#453] [Ideas]
I will love a column type know as serial or Index to be added

The column should work like the regular iteration in for loop that increments after each row.

![image](https://user-images.githubusercontent.com/82326559/170687449-704b64e7-9b1b-402d-b8f8-a5870f2597e1.png)

**Comment by Stridesdata** [comment]
![image](https://user-images.githubusercontent.com/82326559/170689293-b373c909-68e3-4730-b38e-03a8e58bc6e2.png)

**Comment by luanfreitasdev** [comment]
using the loop will reset the value in pagination

**Reply by Stridesdata** [reply]
I don't understand.. Luan

**Reply by Unisolgr14** [reply]
@luanfreitasdev is there a way to stop reseting the value in pagination?

**Comment by rambhosale** [comment]
@Stridesdata do you want to achieve something like this example? https://datatables.net/extensions/fixedcolumns/examples/initialisation/index_column.html

**Comment by Stridesdata** [comment]
Exactly

**Reply by Unisolgr14** [reply]
have you got the solution for this @Stridesdata

**Reply by luanfreitasdev** [reply]
https://v4.livewire-powergrid.com/get-started/release-notes.html#row-index

**Reply by Unisolgr14** [reply]
Thanks for reply Sir @luanfreitasdev . Sadly I am using v3. I fear any upgrades as my project is in a very crucial stage. Any precautions to have a safe upgrade to v4?

**Reply by Unisolgr14** [reply]
Greetings Sir @luanfreitasdev. I have upgraded to v4 (after a year). I have got the row-index now. Thank you for this update in v4 which was not there in v3. But the new issue is it does not get eported in csv. And the row got swap to the left.
